apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/testing/transform.js

```diff
@@ -214,9 +214,9 @@ const transform = {
     },
     prepareCountIssues : (data) => {
       let obj={
-        'High': data['HIGH'] || 0,
-        'Medium': data['MEDIUM'] || 0,
-        'Low': data['LOW'] || 0
+        'High': (data && data['HIGH']) ? data['HIGH'] : 0,
```

Comment:
when will data be null?

Reply:
when from backend, count summary is not coming